distribution/openrct2.d.ts

```diff
@@ -2814,6 +2819,26 @@ declare global {
          * Removes all items from the guest's possession.
          */
         removeAllItems(): void;
+
+        /**
+         * The animations available to this guest.
+         */
+        readonly availableAnimations: string[];
+
+        /**
+         * The animation the guest is currently exhibiting.
+         */
+        animation: string;
```

Comment:
Wouldn't it be nicer if we use a string union here with the possible animation names instead of a blank string?

Reply:
You're right, that would be a good improvement. I'll add it.

Reply:
Done! I've added a `GuestAnimation` and `StaffAnimation` type.